app/models/manageiq/providers/redhat/inventory/parser/infra_manager.rb

```diff
@@ -13,6 +19,10 @@ def parse
     $rhevm_log.info("#{log_header}...Complete")
   end
 
+  def targeted_collection?
+    collector.class.name.include?("TargetCollection")
```

Comment:
Is there any other way to distinguish the collectors?

Reply:
well I could do collector.class == ...::TargetCollection, would that be better?

Reply:
Maybe we could add a method to both